internal/mocks/mock_private_endpoints.go

```diff
@@ -73,10 +73,10 @@ func (m *MockPrivateEndpointDescriber) EXPECT() *MockPrivateEndpointDescriberMoc
 }
 
 // PrivateEndpoint mocks base method.
-func (m *MockPrivateEndpointDescriber) PrivateEndpoint(arg0, arg1, arg2 string) (interface{}, error) {
+func (m *MockPrivateEndpointDescriber) PrivateEndpoint(arg0, arg1, arg2 string) (*admin.EndpointService, error) {
```

Comment:
Moving from interfaces to explicit types in mocks and stores helps us with future maintenance. 
We should move them to atlas